docs/index.md

```diff
@@ -2,86 +2,70 @@
 
-* **A Job**: When something is uploaded onto the **Cluster** and used to run an operation, or when Cluster resources are utilized to run a program, this use of the **Cluster's** resources is called a **Job**.
+When you use your personal computer, the operating system decides what resources (CPU, RAM, etc.) should be used for a given task. For example, if you were to open a calculator program and multiply some numbers, the operating system would determine what CPU threads should be used to perform the calculation, and how much of the graphics card should be used in rendering the calculator on your screen.
 
+An HPC **job scheduler** does the same thing but on a much bigger scale. Instead of a single process on one machine, an HPC cluster handles **jobs** which can be run on many machines at one time. With multiple users, the scheduler must manage incoming requests, reserve the resources, and connect those resources to those who have requested them. The Unity job scheduler is called **Slurm**.
 
 ### How Unity Works ###
-<!-- While you may not need to master every bit of the operation here at the Unity Cluster, it is important that you generally know how the cluster operates, because it can help your troubleshooting in the future. Below is a simplified diagram of the structure of Unity.  -->
 
 Here is a general step by step process that governs how the Unity Cluster works:
 
 ![Unity Diagram](res/unity.png)
 
-1. The client connects to Unity using [SSH](connecting/ssh.md), or the [Jupyter Lab](connecting/jupyter.md). (Bottom right of image)
-    * When you log into the Unity Cluster, you can interact with the Cluster either using Jupyter Lab, or Terminal.
-    * If you are not comfortable working with a Linux SSH terminal interface, it is recommended that you use Jupyter Lab.
-2. Once connected, a job is requested through the scheduler, and the scheduler adds your requests to each *necessary que*.
-    * '*Necessary*' because if the job requested requires 2 GPUs and 1 CPU then you need 2 positions in the GPU que and 1 position in the CPU que. This means that the wait time to get your required resources could be longer, but this doesn't mean the runtime of your job will also be longer.
-3. Once resources are available (cores, gpu, memory, etc.), the scheduler starts your job.
-4. Once your job completes, the result returns to the client.
\ No newline at end of file
+1. Client connects to Unity using [SSH](connecting/ssh.md) or [JupyterHub](connecting/jupyter.md).
+1. Client schedules a job through Slurm.
+1. Slurm allocates resources and connects the client to them.
+1. Job runs.
+    * Interactive job: Client does their work until they close the session or run out of time.
+    * Batch job: The job runs until it finishes its tasks or runs out of time.
+1. Job returns output.
+    * Batch job: The output to the command line is saved by slurm to a file like `slurm-xyz.out`.
```

Comment:
Better might be: "The console output of the commands in the batch script are written to a specified file, or `slurm-###.out`."

Reply:
I would go with slurm-<job ID>.out to help the user identify where the job's output goes (especially after a few of these accumulate) and realize it's not a random number.